Makefile

```diff
@@ -85,13 +85,9 @@ lint-dockerfile:
 	@hack/lint-dockerfile
 .PHONY: lint-dockerfile
 
-.INTERMEDIATE: Dockerfile.dev
-Dockerfile.dev: Dockerfile Dockerfile.centos.patch
```

Comment:
```
i will remove that task.  I don't see the point at the moment
```

can you revert this? we use it so we don't need to build using any of the registry images locally